ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/traffic-policies.j2

```diff
@@ -149,6 +149,36 @@ No Match condition configured.
 {%             endif %}
 
 {%         endfor %}
+{%     endif %}
+{# Traffic-Policy Interfaces #}
+{%     set traffic_policy_interfaces = [] %}
```

Comment:
{%     set traffic_policy_eth_interfaces = [] %}
{%     set traffic_policy_po_interfaces = [] %}

Reply:
Fixed according to Claus' suggestion.